discojs/src/client/decentralized/decentralized_client.ts

```diff
@@ -177,60 +208,69 @@ export class DecentralizedClient extends Client {
    * @param connections 
    * @param round 
    */
-  private receivePayloads (connections: Map<NodeID, PeerConnection>, round: number): void {
+  private receivePayloads (connections: Map<NodeID, PeerConnection>): void {
     connections.forEach(async (connection, peerId) => {
-      let currentCommunicationRounds = 0
       debug(`waiting for peer ${peerId}`);
-      do {
+      for (let _ = 0; _ < this.aggregator.communicationRounds; _++) {
```

Comment:
better to use more readable variable name, not the somewhat standard "ignore" variable name

```suggestion
      for (let r = 0; r < this.aggregator.communicationRounds; r++) {
```